### PR TITLE
Release

### PR DESCRIPTION
**Pull request by chuckha**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**
- Fixes #

**Special notes for your reviewer**:

**Release note**:
```
release-note
```

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Release
+
+## Steps to cut a release
+
+1. Bump the version
+2. Commit and open a pull request.
```
Comment (liztio) [inline docs/release.md]: all full stops or no full stops please!

File: docs/release.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+3. Create a tag after that pull request gets squashed onto master. `git tag -a v0.11.1`
+4. Push the tag. `git push origin refs/tags/v0.11.1`
```
Comment (liztio) [inline docs/release.md]: I have my personal branch set to `origin` and heptio/sonobuoy set to `upstream`. Maybe just `git push --tags`?

File: docs/release.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Release
+
+## Steps to cut a release
+
+1. Bump the version
```
Comment (liztio) [inline docs/release.md]: point out where this gets updated, maybe?

File: docs/release.md (hunk 1 above)
Comment (chuckha) [reply docs/release.md]: true

**Pull request merged**